packages/cpp-debug/package.json

```diff
@@ -14,7 +14,7 @@
   "keywords": [
     "theia-extension", "cpp", "debug"
   ],
-  "version": "0.3.15",
+  "version": "0.3.16",
```

Comment:
hello, shouldn't it be now 0.3.17 ?

Reply:
This PR was made before 0.3.17 was released.  It might be some additional work bumping to 0.3.17 (maybe the APIs have changed?), so I would be fine with merging this and doing 0.3.16 -> 0.3.17 in another patch.